Sources/eppo/EppoClient.swift

```diff
@@ -1,12 +1,13 @@
 import Foundation;
 
 // todo: make this a build argument (FF-1944)
-let sdkVersion = "1.2.1"
+let sdkVersion = "1.3.0"
```

Comment:
This is a breaking API change and some behavioral changes but only subtly.

Reply:
FWIW, according to semver, breaking changes require bumping the major version, so this  should be v2.0.0

Reply:
Yes agreed - I was wondering if we could fudge it a bit but thanks for lending your vote, will bump to 2.0.0. We will be releasing UFC as 3.0.0 and include these changes.